Wikipedia/Code/InsertMediaSettingsViewController.swift

```diff
@@ -224,10 +235,20 @@ final class InsertMediaSettingsViewController: ViewController {
             navigationItem.rightBarButtonItem = nextButton
             navigationController?.navigationBar.topItem?.title = String()
             self.apply(theme: theme)
+
+            loadWikitext { result in
+                switch result {
+                case let .success(wikitext):
+                    self.wikitext = wikitext
+                case let .failure(error):
+                    print("error \(error)")
+                }
+            }
```

Comment:
If you do decide to keep this - can you wrap it up in a `fromImageRecommendations` conditional so we aren't fetching it in the editor context?